app/experimenter/legacy-ui/rapid/__tests__/resultsTable.test.tsx

```diff
@@ -0,0 +1,172 @@
+        metric: "retained",
+        statistic: "binomial",
+        branch: "treatment",
+        point: 0.64215686274509809,
+        lower: 0.57529460650836961,
+        upper: 0.7063786618426765,
+      },
+      {
+        metric: "retained",
+        statistic: "binomial",
+        branch: "treatment",
+        comparison: "difference",
+        point: 0.032060163779913255,
+        lower: -0.065023804214299957,
+        upper: 0.12483606976999304,
+      },
+      {
+        metric: "search_count",
+        statistic: "mean",
+        branch: "control",
+        ci_width: 0.95,
+        point: 14.967359019193298,
+        lower: 10.534758870048162,
+        upper: 20.754349791764547,
+      },
+      {
+        metric: "search_count",
+        statistic: "mean",
+        branch: "treatment",
+        point: 25.456361412643364,
+        lower: 18.998951440573688,
+        upper: 33.549291754637153,
+      },
+      {
+        metric: "search_count",
+        statistic: "mean",
+        branch: "treatment",
+        comparison: "difference",
+        point: 5.0758527676460012,
+        upper: -5.63685604594333,
+        lower: -15.289651027022447,
+      },
+      {
+        metric: "identity",
+        statistic: "count",
+        branch: "control",
+        point: 198,
+      },
+      {
+        metric: "identity",
+        statistic: "count",
+        branch: "treatment",
+        point: 200,
+      },
+    ],
+  },
+  variants: [],
+};
+
+describe("<ResultsTable />", () => {
+  it("has the correct headings", async () => {
+    const EXPECTED_HEADINGS = [
+      "Picture-in-Picture Conversion",
+      "2-Week Browser Retention",
+      "Daily Mean Searches Per User",
+      "Total Users",
+    ];
+
+    await act(async () => {
+      const { getByText } = render(
+        <ResultsTable experimentData={EXPERIMENT_DATA} />,
+      );
+
+      EXPECTED_HEADINGS.forEach((heading) => {
+        expect(getByText(heading)).toBeInTheDocument();
+      });
+    });
+  });
+
+  it("has the expected variant and user count", async () => {
+    await act(async () => {
+      const { getByText } = render(
+        <ResultsTable experimentData={EXPERIMENT_DATA} />,
+      );
+
+      expect(getByText("control")).toBeInTheDocument();
+      expect(getByText("treatment")).toBeInTheDocument();
+      expect(getByText("198")).toBeInTheDocument();
+    });
+  });
+
+  it("has correctly labelled result significance", async () => {
+    await act(async () => {
+      const { getByTestId } = render(
+        <ResultsTable experimentData={EXPERIMENT_DATA} />,
+      );
+
+      const resultsTable = getByTestId("results-table");
+      expect(
+        resultsTable.getElementsByClassName("positive-significance"),
+      ).toHaveLength(1);
+      expect(
+        resultsTable.getElementsByClassName("negative-significance"),
+      ).toHaveLength(1);
+      expect(
+        resultsTable.getElementsByClassName("neutral-significance"),
+      ).toHaveLength(1);
```

Comment:
Any reason not to use test-id for these? `expect(getByTestId("neutral-significance")).toBeInTheDocument()` is a little more succinct. Not a blocker though.

Reply:
I was trying to reduce the number of test ids I added in. But I suppose having 3 instead of 1 is no big deal!

Reply:
Alternatively you could query by content/text. Whenever I get concerned about testids I am reminded of [this](https://github.com/testing-library/react-testing-library/issues/479#issuecomment-529548194), and specifically 1) they add very little overall weight to the code, and 2) they can be leveraged in E2E testing should we decide to have it.